lib/ParamediciOSPermissions.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+/* jshint node: true */
+var path     = require('path');
+var fs       = require('fs');
+var shelljs  = require("shelljs");
+var logger   = require('./utils').logger;
+var util     = require('./utils').utilities;
+
+var TCC_FOLDER_PERMISSION = 0777;
```

Comment:
Why doesn't `0644` work? If execution permissions are required on the folder, then please use `0755`. Using `0777` is a security hazard.